### DISCUSSION TITLE
Condensing Text Output to Fewer Lines

### DISCUSSION THREAD

**Discussion by Soeroah** [Purfview/whisper-standalone-win#311] [Q&A] [answered]
Hi, sorry for the awkward way of phrasing this. I'm not quite able to parse the more technical terms so I dunno what to look for/call things.

I'm trying to have my text file output include longer stretches of words on each line, if I can. I usually use this program to transcribe conversations, but the output is currently returning some lines with one to three words each. Some sentences of less than 20 words are split across numerous lines.

Is there a parameter that can either force the final .text output to be a block (I'd rather go through and add a new line at the end of each sentence than have to backspace/delete almost every line to get a full sentence) or to change the timestamped chunks being transcribed to be longer than what seems to be about 2 seconds per line? I tried --sentences but I don't think it's compatible with output_format=text

Thanks

**Comment by Purfview** [comment, marked as answer]
> I'm trying to have my text file output include longer stretches of words on each line, if I can. 

Use `--sentence`, this is for `srt` output. if you need clean text then use program like Subtitle Edit to open srt and save it to whatever you want.
